cocos/ui/editbox/edit-box.ts

```diff
@@ -479,9 +479,9 @@ export class EditBox extends Component {
     /**
      * @deprecated since v3.5.0, this is an engine private interface that will be removed in the future.
      */
-    public _editBoxEditingDidEnded () {
+    public _editBoxEditingDidEnded (text?: string) {
         ComponentEventHandler.emitEvents(this.editingDidEnded, this);
-        this.node.emit(EventType.EDITING_DID_ENDED, this);
+        this.node.emit(EventType.EDITING_DID_ENDED, this, text);
     }
```

Comment:
```diff
/**
 * @deprecated since v3.5.0, this is an engine private interface that will be removed in the future.
+* @param text  (some descriptions)
 */
```

Can we add some descriptions to decribe what this text for 
and for now only ByteDance platform has this param

Reply:
Done.